src/ResourceManager/Profile/Commands.Profile/Commands.Profile.csproj

```diff
@@ -11,8 +11,6 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
Looks like there are some unintended changes. Can you please revert changes to these files?